bark_ml/library_wrappers/lib_tf2rl/runners/gail_runner.py

```diff
@@ -6,58 +10,51 @@
 import tensorflow as tf
 tf.compat.v1.enable_v2_behavior()
 
-# tf2rl imports
-import tf2rl
-from tf2rl.experiments.irl_trainer import IRLTrainer
-from tf2rl.experiments.utils import restore_latest_n_traj
-
-# BARK-ML imports
-from bark_ml.library_wrappers.lib_tf2rl.runners.tf2rl_runner import TF2RLRunner
 
 class GAILRunner(TF2RLRunner):
   """GAIL runner implementation based on tf2rl library."""
 
   def __init__(self,
-              environment=None,
-              agent=None,
-              params=None,
-              expert_trajs=None):
+               environment=None,
+               agent=None,
+               params=None,
+               expert_trajs=None):
     """initializing a TF2RL GAIL agent.
     Args:
       - environment: BARK runtime or open-ai gym type environment.
       - agent: TF2RL GAIL agent
       - params: BARK ParameterServer
       - expert_trajs: Expert trajectories.
     """
-    
+
     self._expert_trajs = expert_trajs
     if not self._expert_trajs:
       import numpy as np
       self._expert_trajs = {
-          'obses': np.empty([0,0]),
-          'next_obses': np.empty([0,0]),
-          'acts': np.empty([0,0])}
-    
+          'obses': np.empty([0, 0]),
+          'next_obses': np.empty([0, 0]),
+          'acts': np.empty([0, 0])}
+
     TF2RLRunner.__init__(self,
-                    environment=environment,
-                    agent=agent,
-                    params=params)        
+                         environment=environment,
+                         agent=agent,
+                         params=params)
 
   def _train(self):
     """Traines the GAIL agent with the IRLTrainer which was
     instantiated by self.GetTrainer.
     """
     self._trainer()
 
-  
   def GetTrainer(self):
     """Creates an IRLtrainer instance."""
     policy = self._agent.generator   # the agent's generator network, so in our case the DDPG agent
-    irl = self._agent.discriminator  # the agent's discriminator network so in our case the GAIL network
+    # the agent's discriminator network so in our case the GAIL network
```

Comment:
Maybe not so nice because it's inconsistent.

Reply:
I see, but this is the standard. I will now merge it as we can train on the dataset now and this should be our focus.